packages/util/src/types.ts

```diff
@@ -141,3 +141,10 @@ export function toType<T extends TypeOutput>(
     return `0x${output.toString('hex')}` as TypeOutputReturnType[T]
   }
 }
+
+/**
+ * An object containing pairs of keys and the RLP encoded value for that key
+ */
+export type RlpValues = {
+  [value: string]: Buffer | undefined
+}
```

Comment:
Just as some first read comment: this is a *very*  central place for such a new type which will become part of the overall API/functionality set if we introduce here. Not against it or anything and also haven't made up my mind, @ryanio what do you think (others as well of course), introduce here or not?

Alternative would be to do a bit more locally e.g. also in the `util.ts` file from the tx library.

Reply:
Now that you mention it, it probably does make sense to keep it inside `tx`.  I can't envision a case where a helper like this is going to be used outside of the specific use case we're looking at.  I'll move it.

Reply:
have inlined it into the function method in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1568/commits/be2bb7c68f647dfe9964ad6eb547d502441d0aa4 so the type is no longer needed